frontend/src/modules/Datasets/components/DatasetGovernance.js

```diff
@@ -38,41 +39,36 @@ export const DatasetGovernance = (props) => {
           {dataset.stewards}
         </Typography>
       </CardContent>
-      <CardContent>
-        <Typography color="textSecondary" variant="subtitle2">
-          Auto-Approval
-        </Typography>
-        <Box sx={{ mt: 1 }}>
-          <Label color="primary">
-            {dataset.autoApprovalEnabled ? 'Enabled' : 'Disabled'}
-          </Label>
-        </Box>
-      </CardContent>
-      <CardContent>
-        <Typography color="textSecondary" variant="subtitle2">
-          Classification
-        </Typography>
-        <Box sx={{ mt: 1 }}>
-          <Label color="primary">{dataset.confidentiality}</Label>
-        </Box>
-      </CardContent>
-      <CardContent>
-        <Typography color="textSecondary" variant="subtitle2">
-          Topics
-        </Typography>
-        <Box sx={{ mt: 1 }}>
-          {dataset.topics &&
-            dataset.topics.length > 0 &&
-            dataset.topics.map((t) => (
-              <Chip
-                sx={{ mr: 0.5, mb: 0.5 }}
-                key={t}
-                label={t}
-                variant="outlined"
-              />
-            ))}
-        </Box>
-      </CardContent>
+      {isFeatureEnabled('datasets', 'confidentiality_dropdown') && (
```

Comment:
Do not think we should be removing Auto Approval value here from UI

Reply:
Thanks for catching. Adding it back again!